Pycloud Deployer/infra_classes.py

```diff
@@ -0,0 +1,288 @@
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.compute import ComputeManagementClient
+from modules import (
+    create_dbserver_parameters, 
+    create_publicip_parameters, 
+    create_nic_parameters, 
+    create_webserver_parameters, 
+    run_command_parameters
+)
+
+class WebServer():
+    def __init__(self, subscription_id, token_credential, deployment_name) -> None:
+        """Initiate the WebServer class properties
+        """
+
+        # Creating management clients
+        self.rss_client = ResourceManagementClient(token_credential, subscription_id)
+        self.net_client = NetworkManagementClient(token_credential, subscription_id)
+        self.compute_client = ComputeManagementClient(token_credential, subscription_id)
+
+        # Declare instance variables which serve as global variables within the class
+        self.location = "southafricanorth"
```

Comment:
Consider making these configurable through parameters or a configuration file for flexibility.